alan_workspace/files.txt

```diff
@@ -0,0 +1,21 @@
+Files that relate to this task. It can be:
+- Edited files
+- References files
+
+You can comment:
+- Start with prefix `retrieve` if you want to retrieve files that you want. You can explain why do you want alan retrieve these files to improve the result. This mode only support append new files.
+  - Example 1: retrieve test_order.go to add unit test
+  - Example 2: retrieve all files in folder db to update the connection string
+- Start with prefix `vretrieve` if you want that alan will retrieve with verification. This mode support append and remove files.
+  - Example 1: vretrieve test_order.go to add unit test
+  - Example 2: vretrieve remove files in folder config because this isn't related to this task
+
+This is files list, please verify these files carefully:
+```
+src/dispatch/api.py
+src/dispatch/team/service.py
```

Comment:
alan vretrieve file in folder src/dispatch/organization as example to write code